### ISSUE TITLE
"Slide-up" UI

### ISSUE THREAD

**Issue by RomanHdz** [rpetrich/ProSwitcher#197]
Hello!

I love ProSwitcher, and after messing around with the application I got a couple ideas for the UI. Here's a link to a mockup image I made:

http://web.me.com/romanhernandez/ProSwitcher_Ideas/UI.html

Firstly, perhaps instead of having the current application do a "close" animation, a sheet could slide up with all your open applications like an overlay.  It may also make the presentation of the cards better if there was a small drop-shadow beneath each card, as well as making the two cards on the left and right of the focussed app faded a bit.

Instead of page indicators, the app icon could be displayed (I believe this was already mentioned).  The focussed app should have its icon a bit bigger than the rest, to make it stand out a bit more.

My final (and favorite) idea is a bar on the bottom of the screen that can display remaining RAM or anything else that may be useful.  Also, tapping the bar makes ProSwitcher slide back down.

I hope you like them :)

**Comment by GabsNL** [comment]
I LOVE allmost all these ideas! I didn't quite understood what you meant with the alt closing animation? The dropshadowlessness annoyed me without even realizing what it was, thanks for puting it out there. Really nice mockup, I can honestly say all the ideas are great. Duno' if the bottombar will be implented, Ryan ones said that he doesn't want buttons/bars etc. But then again, who knows! These guys listen to their community :)

**Comment by rpetrich** [comment]
I don't think we will be implementing a "slide up sheet"... it just changes way too many things from a code perspective.

Drop-shadows are planned, but only if we can implement them while keeping performance up (and not sacrifice some other features we have planned ;)

App-icons as page indicators has been proposed before--you've refined this idea slightly. I like it, but it will definitely have to be an option.

Finding a way to show RAM usage especially as a percentage of "app RAM usage" could be very helpful--we will accept patches in this regard, but since this feature requires root it won't be simple.
